### PR TITLE
feat(app-toolbar): updated filesystem/sync icons

### PR DESCRIPTION
**Pull request by juniusfree**
@shanberg I updated the [sync icons](https://www.craft.do/s/MAGVSD9LgwXSnF/b/3BF560E4-65B3-40F9-9F30-289176B28928/Better_Accessibility). Let me know your thoughts. Thanks.

Synced
image
image

Syncing
image
image

Error
image
image

Database modal
image
image

App toolbar right
image
image

### CODE REVIEW

**Comment by shanberg** [comment]
This is a huge improvement in readability.

I don't think there's a clear answer to what material-ui icon best represents a knowledge graph database, so I won’t dwell too much on that change, but we should use the same icon wherever the same concept is being represented.

Relatedly, this icon/status indicator can replace the filesystem dialog button icon, and the database icon in the filesystem dialog itself.

**Comment by juniusfree** [comment]
I'll try to see if there is a more relevant icon that'll represent the graph db and use it in the filesystem.

**Comment by juniusfree** [comment]
@shanberg I can't find other icons for db so I just use Storage icon for the filesystem and just combine it with the sync status icon.

**Comment by tangjeff0** [comment]
Is this ready for review? @shanberg can you review too?

File: src/cljs/athens/views/app_toolbar.cljs [hunk 1]
```diff
@@ -179,9 +165,28 @@
                      :title "Open Settings"
                      :active   (= @route-name :settings)}
              [:> Settings]]
+
             [button {:on-click #(dispatch [:modal/toggle])
                      :title "Choose Database"}
-             [:> LibraryBooks]]
+             [:div {:style {:display "flex"}}
```
Comment (shanberg) [inline src/cljs/athens/views/app_toolbar.cljs:171]: This sequence of added divs and styles could possibly be refactored into a single style applied to the button itself, and using `::stylefy/manual [[:div {:display "flex"}...]] to style its children.

File: src/cljs/athens/views/app_toolbar.cljs [hunk 2]
```diff
@@ -179,9 +165,28 @@
...
+              [:> Storage {:style {:align-self "center"}}]
+              [:div {:style {:margin-left "-10px"
+                             :align-self "flex-end"}}
+               (cond
+                 (= @socket-status :closed)
+                 [:> Error (merge (use-style sync-icon-style)
```
Comment (shanberg) [inline src/cljs/athens/views/app_toolbar.cljs:177]: A possibly cleaner way to change the colors according to status would be to add a class on the button, `.closed`, `.running` etc., and change the color in the style object.

**Review by shanberg — APPROVED** [review]
Beautiful. Really ties the toolbar together.

**Review by tangjeff0 — APPROVED** [review]

**Pull request merged**